battle_abilities_test.go

```diff
@@ -6,7 +6,30 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("Scrappy", func() {
+var _ = Describe("Ability: Air Lock", func() {
+	a := Agent(new(dumbAgent))
+
+	It("should negate the effects of weather", func() {
```

Comment:
Can we have a test to see if the effect gets removed when the pokemon is removed from the battlefield?